src/main/java/randoop/main/GenInputsAbstract.java

```diff
@@ -171,6 +171,13 @@ public GenInputsAbstract(
   @Option("What to do if a flaky test is generated")
   public static FlakyTestAction flaky_test_behavior = FlakyTestAction.HALT;
 
+  /**
+   * How many suspected flaky methods to print out (sorted by likelihood) upon flaky test
+   * generation. A flaky test is one that behaves differently on different executions.
+   */
+  @Option("Maximum number of suspected flaky methods to print")
+  public static int num_suspected_flaky_test_to_output = 20;
```

Comment:
Should `_test_` be `_method_`?  It's not tests, but methods from user code, that are being output, right?
Also, 20 seems like a large number.  Are the tool's guesses so poor that a large number are usually needed?